examples/rbd/storageclass.yaml

```diff
@@ -42,21 +42,17 @@ parameters:
    csi.storage.k8s.io/fstype: ext4
    # uncomment the following to use rbd-nbd as mounter on supported nodes
    # mounter: rbd-nbd
-
    # Prefix to use for naming RBD images.
    # If omitted, defaults to "csi-vol-".
    # volumeNamePrefix: "foo-bar-"
-
    # Instruct the plugin it has to encrypt the volume
    # By default it is disabled. Valid values are "true" or "false".
    # A string is expected here, i.e. "true", not true.
    # encrypted: "true"
-
    # Use external key management system for encryption passphrases by specifying
    # a unique ID matching KMS ConfigMap. The ID is only used for correlation to
-   # config map entry.
+   # configmap entry.
    # encryptionKMSID: <kms-config-id>
-
```

Comment:
why new lines are removed in both storageclass?

Reply:
We could keep as it is too.. Let me revert this change.

Reply:
@Madhu-1 done.